### DISCUSSION TITLE
Mapping 500 $a

### DISCUSSION THREAD

**Discussion by laura-ake** [crystalyragui/MARC2RDA#330] [General]
Before I make my first commit, want to run the justification by you.  500 is a general note that could relate to Work, Expression, Manifestation, Item, possibly a combination of 2 or more. as well as possibly relating to Agents and other aspects of these.  A generalized mapping would have to take this into account.
  
RDA Vocabularies has this:  [rdaxLP00015](http://www.rdaregistry.info/Elements/x/#P00015) 

CURIE  rdax:P00015         
Label            "has note on RDA entity"
Definition    "Relates an RDA entity to a broad unstructured description of one or more attributes of an RDA entity."	    
Subproperty of   (blank)

Each property in the element set:
has a domain of the class that represents the RDA Entity entity.
is linked from its child datatype property in [RDA Entity datatype properties](http://rdaregistry.info/Elements/x/datatype/) by rdfs:subPropertyOf.
is linked from its child object property in [RDA Entity object properties](http://rdaregistry.info/Elements/x/object/) by rdfs:subPropertyOf.  
         

I am thinking - 

Could I use this property with the RDA Entity subclass Manifestation?  (i.e. is it "inheritable").  Something tells me it can't be that simple.
 
If not, could I establish an RDA Entity (give it a URI), 
make it the subject of a P00015 property, the object being the note text string, 
and relate at least one other part of the description to it using, e.g., the Manifestation property "has related RDA entity of manifestation " ([rdam:P30264](http://rdaregistry.info/Elements/m/P30264))?  

This assumes that the note has something to do with some RDA entity.  

The relation of the 500 note to the manifestation is that it was added to a MARC record that primarily describes that manifestation - but we don't know enough to make a direct relation to the manifestation.

If this is a logical use of RDA vocabs,  is it too unorthodox?  Will systems know what to do with RDA entity relations of this type?  

-----------------------------
The only alternative I see would be to do a "best guess" mapping to a "has note about" property for one of the other Entity types, and I'd pick Manifestation as a best guess since more of the MARC records overall are "about" Manifestations than other elements.   A certain number of these are going to be wrong amongst the population of 500 notes.

**Comment by CECSpecialistI** [comment]
@laura-ake This is such a hard one to conceptualize outside a MARC context! The 500, when it's written in MARC, has to be attached to...something. The "thing" being cataloged. Using "has note on RDA entity" still leaves us with the best-guess problem of deciding which RDA entity a given note applies to. I would think you could use this property with any RDA Entity, including a Manifestation, but it would make more sense in that case to use the more specific property for notes on manifestations. I'm leaning towards "has note on manifestation" but this could use some more input from the rest of the group.

**Comment by CECSpecialistI** [comment]
[Entity=Description set based on source MARC record] has note on RDA Entity [500]
[Entity=Description set based on source MARC record] has related RDA Work [Work description]
?
Examples from RDA?

**Reply by CECSpecialistI** [reply]
We decided to think about this and discuss it more at the meeting on 2022-02-23. @laura-ake @CECSpecialistI @lake44me @gerontakos @AdamSchiff @Soersatie @JianPLee @junghaelee

**Comment by CECSpecialistI** [comment]
Push out to March 16, with more investigation by everyone after next week.

**Comment by CECSpecialistI** [comment]
What if we used an unconstrained and very-broad property here, "has note on resource", to eliminate guesswork and just attach the note to each entity being described? The property I'm thinking of is [rdau:P60470](http://rdaregistry.info/Elements/u/P60470). It's really vague, which reduces its usefulness (I cringe at the word "resource" almost as much as "miscellaneous"), but it would be appropriate to any 500 $a and I don't know what else could be.

**Reply by lake44me** [reply]
I like the property "has note on resource" for 500 rather than "has note on entity".  But if rdau:060470 is the property, what is the subject of the triple?  It would need to have a URI, and to relate to the Work, Expression, and Manifestation, right?  

There are properties to do that for RDA Entity

<https://URIforanRDAentity>
  a http://rdaregistry.info/Elements/x/P00001;    [is an RDA Entity]
  http://rdaregistry.info/Elements/x/P00015  "...This book has the same title as the more controversial pamphlet published in the same year; both authors are listed as 'anonymous' but are distinct persons, according to the bookseller Peaslee's memoirs... -- Jackson, j. W.  Notes on the Reformation.  Edinburgh, Wilson & Co., 1919.";   [has note on RDA Entity]
  [http://rdaregistry.info/Elements/x/P00015 https://URIforanRDAWork      [has related work of RDA entity] 

etc. etc. for the related RDA Expression and Manifestation 

in the "description set".  (Can a "description set" be an entity?  What class of entity would it be?)

I mean, we can throw an unconstrained property into a triplestore but if it doesn't relate/link to one of the RDA entities, it's not much good.  

'Scuse my crummy attempt at Turtle.

**Reply by CECSpecialistI** [reply]
Right, I was thinking the predicate and object would be attached to all the entities being described, the work, expression, manifestation, and item. RDA Entity property has a domain of RDA entity, right? Whereas unconstrained properties do not have domains? so it would be 

https://URIforRDAWork rdau:P60470 ""...This book has the same title as the more controversial pamphlet published in the same year; both authors are listed as 'anonymous' but are distinct persons, according to the bookseller Peaslee's memoirs... -- Jackson, j. W. Notes on the Reformation. Edinburgh, Wilson & Co., 1919."
https://URIforRDAExpression rdau:P60470  ""...This book has the same title as the more controversial pamphlet published in the same year; both authors are listed as 'anonymous' but are distinct persons, according to the bookseller Peaslee's memoirs... -- Jackson, j. W. Notes on the Reformation. Edinburgh, Wilson & Co., 1919."
and so on for the expressions and items. 

Now that I write that out, it sounds silly. Why not use "has note on work" for works and "has note on expression" for expression etc. I shy away from that approach because it might not actually be a note on the work. Calling it a resource makes it seem like "hey this could be related maybe". Trying to pick out which 500 notes apply to what entities sounds like a nightmare and I can't think of better solutions. These are really suited for a "record" environment rather than a linked data one.

The notes aren't about the description set...so ... Would the RDA Entity here be a description set?

**Comment by AdamSchiff** [comment]
Unconstrained properties are not considered RDA-conformant (see https://access.rdatoolkit.org/Guidance/GuidanceById/71f300b9-18e4-495a-969b-4c59941b33a4).  While we could map to unconstrained properties found in the RDA Registry (such as "has note on resource" http://rdaregistry.info/Elements/u/P60470), the RSC would not consider this well-formed RDA data.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by CECSpecialistI** [reply]
That makes a lot of sense. Using the appropriate properties for each entity type is obviously better than using a less specific property. Hmm. These pesky 500 notes. Should they just be applied at the manifestation level? Relying on relationships between works/expressions/manifestations/items to contextualize them? Repeating them for every entity seems like poorly-formed RDA as well :/

**Reply by lake44me** [reply]
I am willing to map it for now to Manifestation entity as being the **most likely to be the correct entity**, but I would like this to be raised as a question with RSC.  DECISION?

The question:  
Is there another way to map notes or other data where the subject entity of a statement is ambiguous, so that they have a place in the description set?  If so, If so I'd prefer to put them there, because it will call them out as needing review, but they will still be available as data.  Down the road, where we have big pots (like OCLC or various consortia) of RDF metadata, some of which was original cataloging in linked data and other of which is converted from MARC, there isn't likely to be a way to flag converted fields of this nature.  At least, the way described in RDA Data Provenance (https://access.rdatoolkit.org/Resource/Index?externalId=en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac) implies that in a structured description, an identifier (URI?) would need to be made for a statement as a "metadata work" in order to have a statement about it.  
@AdamSchiff is that a clear statement of the problem?  
@gerontakos @Soersatie @junghaelee @JianPLee

**Comment by CECSpecialistI** [comment]
We need a label for questions to raise to RSC, don't we? Or an ongoing list of questions?

**Comment by lake44me** [comment]
For this, I think a label would be fine. Just so we can find and compile them later

Get Outlook for Android<https://aka.ms/ghei36>
________________________________
From: Crystal Clements ***@***.***>
Sent: Wednesday, March 23, 2022 1:05:31 PM
To: uwlib-cams/MARC2RDA ***@***.***>
Cc: Laura Akerman ***@***.***>; Mention ***@***.***>
Subject: [External] Re: [uwlib-cams/MARC2RDA] Mapping 500 $a (Discussion #330)


We need a label for questions to raise to RSC, don't we? Or an ongoing list of questions?

**Comment by GordonDunsire** [comment]
Hi all

This issue was partially considered by the 3R Project.

The context was the legacy element 'note on frequency' which had Manifestation as domain. LRM shifted frequency to be a diachronic Work attribute, so we had to fully deprecate the Manifestation element and accommodate its instructions under 'note on work' and the more complex structures for describing diachronic works. We did, however, agree that it would not be necessary to make the label disappear, so it was added as a 'use' reference to 'note on manifestation'.

All specific note elements are soft-deprecated in favour of the general note element for the relevant entity because there is no utility in constraining the semantics of unstructured strings that were useful only for general keyword indexing and retrieval in addition to the primary user task of 'identify'.

We decided that the content of a previous note on frequency would not be out of place as a general note in a description set for a manifestation of a diachronic work because of the WEM-lock. But this implies a very relaxed approach to the semantics for all manifestations; a note can be about anything considered relevant (the manifestation being described, any other entity associated with the manifestation, the description of the manifestation itself that is really data provenance, and anything else that is (subjectively) relevant). Of course, it is the same in general for 'note on item'.

I see little distinction in the insignificant impact on general keyword indexes in general applications between mapping 500 to rdam:P30137 or rdax:P00015.

However, the 'identify' task will be much better served with a mapping to rdam:P30137. As well as the loose semantics indicated above, in most implementation scenarios the end-user will see an integrated display for a 'resource' description at the terminus manifestation of the search. Otherwise, the contents of rdax:P00015 will need to be pulled-down, but how?

In a linked data scenario, the semantics are primarily dictated by the AAA principle and the Open World Assumption, as with all string data, so instances of "miss" links are tolerable.

There is a simple approach to data provenance for converted fields, using [this option](https://access.rdatoolkit.org/en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac/div_dtm_qgr_vfb).

For example:

<ex:1 rdam:P30137 "Metadata rules!"> rdaw:P10406 "Uwlib conversion (2022-01-01)" .

Hope this helps. It will be interesting to see what RSC says, if you go ahead and ask.

**Comment by CECSpecialistI** [comment]
map to note on manifestation. decision 2022-04-20

**Comment by lake44me** [comment]
I am now adding a condition to apply this mapping only if no subfield 5 exists in the tag.

**Comment by laura-ake** [comment]
I'm afraid this might end up being the "we need to ask Gordon" questions.  Description set is a Thing, but there's nothing in the RDA Vocabularies (she hastily does a search) that links to how to express statements at the Description Set level, other than this:  https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac#section_rdaId_section_gr4_l4l_5cb on Data Provenance.  As I interpret it, on creates a Work for the Description Set (or a Statement) and then can make statements about it.  But whether one can add a note that relates in a non-specific way to one of the Entities described in a Description Set...  is not spelled out, nor how the WEMI entities are related to the set.

Lura

From: Crystal Clements ***@***.***>
Sent: Tuesday, March 22, 2022 6:22 PM
To: uwlib-cams/MARC2RDA ***@***.***>
Cc: Akerman, Laura ***@***.***>; Mention ***@***.***>
Subject: [External] Re: [uwlib-cams/MARC2RDA] Mapping 500 $a (Discussion #330)


Right, I was thinking the predicate and object would be attached to all the entities being described, the work, expression, manifestation, and item. RDA Entity property has a domain of RDA entity, right? Whereas unconstrained properties do not have domains? so it would be

https://URIforRDAWork rdau:P60470 ""...This book has the same title as the more controversial pamphlet published in the same year; both authors are listed as 'anonymous' but are distinct persons, according to the bookseller Peaslee's memoirs... -- Jackson, j. W. Notes on the Reformation. Edinburgh, Wilson & Co., 1919."
https://URIforRDAExpression rdau:P60470 ""...This book has the same title as the more controversial pamphlet published in the same year; both authors are listed as 'anonymous' but are distinct persons, according to the bookseller Peaslee's memoirs... -- Jackson, j. W. Notes on the Reformation. Edinburgh, Wilson & Co., 1919."
and so on for the expressions and items.

Now that I write that out, it sounds silly. Why not use "has note on work" for works and "has note on expression" for expression etc. I shy away from that approach because it might not actually be a note on the work. Calling it a resource makes it seem like "hey this could be related maybe". Trying to pick out which 500 notes apply to what entities sounds like a nightmare and I can't think of better solutions. These are really suited for a "record" environment rather than a linked data one.

The notes aren't about the description set...so ... Would the RDA Entity here be a description set?

-

**Reply by CECSpecialistI** [reply]
I thought we had already decided to just map these to note on manifestation? This email I sent back in March was pretty half-baked, and the decision we made to map to note on manifestation was in April. Do we need to revisit?
